index.html

```diff
@@ -12,7 +12,7 @@
       <nav id="nav-bar">
```

Comment:
You have a good idea, but may I suggest considering adding classes directly to the semantic nav tag instead of adding a class to an ID? This would be a more effective way of adding styles to your navigation menu.